$OpenBSD: patch-payment_py,v 1.1 2019/08/02 11:18:44 semarie Exp $
backport for issue8550 https://hg.tryton.org/modules/account_payment_sepa/rev/217d73cb3e16
Use move_origin instead of origin to generate sepa_remittance_information

Index: payment.py
--- payment.py.orig
+++ payment.py
@@ -309,8 +309,8 @@ class Payment(metaclass=PoolMeta):
     def sepa_remittance_information(self):
         if self.description:
             return self.description
-        elif self.line and self.line.origin:
-            return self.line.origin.rec_name
+        elif self.line and self.line.move_origin:
+            return self.line.move_origin.rec_name
 
     @property
     def sepa_bank_account_number(self):
